command.go

```diff
@@ -87,6 +88,16 @@ func (c *Command) SetEnvVar(key, value string) *Command {
 	return c
 }
 
+// SetSeparateProcessGroup sets whether the command should be run in a separate
+// process group. This is useful to avoid propagating signals from the app process.
```

Comment:
Can you add a note in the comment that this will only work with windows and Unix-based OS?

Reply:
done!